src/client/pages/RetroCertsConfirmationPage/index.js

```diff
@@ -25,6 +25,14 @@ function RetroCertsConfirmationPage(props) {
     return <Redirect to={routes.retroCertsWeeksToCertify} />;
   }
 
+  // For historical reasons the confirmation code is stored as a uuidv4
+  // hash in the database, but we display only the last 7 characters to the user.
```

Comment:
Can you add to the change description or the github issue how to came up with 7 characters and what the probability of a duplicate with another user is?

Reply:
OMG, that's hilarious. _NUMBER really doesn't look like a truncated string! Yes I'll add that to the issue now.

Reply:
Done https://github.com/cagov/unemployment/issues/450